app/models/middleware_messaging.rb

```diff
@@ -0,0 +1,6 @@
+class MiddlewareMessaging < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :middleware_server, :foreign_key => "server_id"
```

Comment:
is this also correct in domain mode? (that jms topic/queue belongs to only one server) @Jiri-Kremser @pilhuhn